modules/oe_translation_poetry/README.md

```diff
@@ -2,14 +2,61 @@
+## Requirements
+
+If you want to use this submodule, you need to add the following packages to your site:
+
+* ec-europa/oe-poetry-client:dev-master
+
+## Setup
+
+### DGT request
+
+To integrate with Poetry, a request has to made to register the site with DGT. See [here](https://github.com/ec-europa/platform-dev/tree/release-2.5/profiles/common/modules/features/nexteuropa_dgt_connector/#requesting-access-to-the-dgt-connector) more information about this. Note: This documentation needs an update.
+
+### Configuration
+
+There are a number of required configuration options needed for the Poetry integration to work. These can be added at this path: `admin/tmgmt/translators/manage/poetry`:
+
+* Poetry service URL - the Poetry endpoint. For the acceptance environment this is `http://intragate.test.ec.europa.eu/DGT/poetry_services/components/poetry.cfc?wsdl`
+* Identifier code - Unless requested otherwise, this can be `WEB`
+* Request title prefix - Unless requested otherwise, this can be `EWCMS`
+* Site ID - The site ID agreed with DGT
+* Application reference code - Unless requested otherwise, this can be `FPFIS`
+* Contact information - the EULogin user account of the default user to be used as a contact for making translations. These are overridable per individual translation request.
+* Organisation information - Information about the DG responsible for the site. For DIGIT sites it can be, and unless otherwise requested by DGT:
+  * Responsible - DIGIT
+  * Author - IE/CE/DIGIT
+  * Requester - IE/CE/DIGIT
+  
+### Environment variables
+
+There are a few settings that need to end up in the `settings.php` file of the site. The values of these are, however, populated as environment variables by Devops.
+
+Using the runner, ensure that the following ends up in the site's `settings.php` file:
 
 ```
-$config['tmgmt.translator.poetry']['settings']['service_wsdl'] = 'http://localhost:8080/build/poetry-mock/wsdl';
+$settings["poetry.identifier.sequence"] = getenv('POETRY_IDENTIFIER_SEQUENCE');
+$settings["poetry.service.username"] = getenv('POETRY_SERVICE_USERNAME');
+$settings["poetry.service.password"] = getenv('POETRY_SERVICE_PASSWORD');
+$settings["poetry.notification.username"] = getenv('POETRY_NOTIFICATION_USERNAME');
+$settings["poetry.notification.password"] = getenv('POETRY_NOTIFICATION_PASSWORD');
 ```
 
-## Requirements
+Then, request Devops to fill those variables on the server with the following values:
 
-If you want to use this submodule, you need to add the following packages to your site:
+* `POETRY_IDENTIFIER_SEQUENCE` - `EWCMS_SEQUENCE`
+* `POETRY_SERVICE_USERNAME` - Username provided by DGT or CEM for accessing the Poetry environment.
+* `POETRY_SERVICE_PASSWORD` - Password provided by DGT or CEM for accessing the Poetry environment.
+* `POETRY_NOTIFICATION_USERNAME` - Username agreed with DGT for authenticating requests made by Poetry to the site (the notifications).
```

Comment:
For notifications DGT doesn't need to be informed neither to agree with the user nor password.
This information will be sent in the XML of the request and used later by Poetry.

I propose something like we have in NextEuropa:

drupal (user/password) for notifications - in lowercase, limited to 15 characters and different to poetry credential. It must identify uniquely the platform where the translations have to be delivered to. Example: projectname-test/projectname-test or projectname-prod/projectname-prod